lambda/storeSimInformationWirelessLogic.ts

```diff
@@ -73,11 +76,20 @@ const h = async (event: SQSEvent): Promise<void> => {
 				wirelessLogicDataLimit,
 			})
 			if ('error' in simDetails) {
+				if (simDetails.error instanceof SIMMissingFromVendorAPI) {
+					await putSimDetailsFunc({
+						iccid,
+						simExisting: false,
+						simDetails: undefined,
+						SIMMissingFromVendorAPI: true,
```

Comment:
Add a `ttl` that expires this state after 7 days, in case the SIM is added to the vendor API later.